src/components/orders/DetailsTable/index.tsx

```diff
@@ -83,6 +84,12 @@ export function DetailsTable(props: Props): JSX.Element {
             <td>From</td>
             <td>{owner}</td>
           </tr>
+          {!partiallyFillable && (
+            <tr>
+              <td>Transaction hash</td>
+              <td>{txHash || '-'}</td>
```

Comment:
It would be nice to add already the Link, isn't it?

Reply:
Yes, but they are in separated branches. Will add after https://github.com/gnosis/gp-ui/pull/184 is merged

Reply:
Yep, I saw it after that PR..